### PR TITLE
Correct assignment for logits in classifier example

### PR DESCRIPTION
**Pull request by davidefiocco**
I tried to address https://github.com/huggingface/pytorch-pretrained-BERT/issues/76
should be correct, but there's likely a more efficient way.

### CODE REVIEW

**Comment by thomwolf** [comment]
Ok thanks, that should work for now. I simplified the output of the classes indeed (only send back loss when a label is provided) so this example broke.
